src/test/java/com/google/api/codegen/testdata/discoveries/php/php_storage.v1.json.baseline

```diff
@@ -1126,7 +1126,14 @@ $requestBody = new Google_Service_Storage_ComposeRequest($client);
 // TODO: Assign values to desired properties of the `requestBody` object.
 
 
-$response = $service->objects->compose($destinationBucket, $destinationObject, $requestBody);
+$optParams = array();
+
+// TODO: Use `'alt' => 'media'` to download media content.
+//
+// $optParams['alt'] = 'media';
```

Comment:
As in the node comments above, if there' s commented -out code the user might want to use, I slightly prefer being more concise by removing the blank line above and indenting the commented-out code.